backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/controller/DashboardController.java

```diff
@@ -0,0 +1,50 @@
+package com.chinguv45tier3team53.tripplannerrest.controller;
+
+import com.chinguv45tier3team53.tripplannerrest.dao.UserRepository;
+import com.chinguv45tier3team53.tripplannerrest.entity.User;
+import com.chinguv45tier3team53.tripplannerrest.service.JwtService;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/dashboard")
+@RequiredArgsConstructor
+public class DashboardController {
+    private final UserRepository repository;
```

Comment:
it's better to have the controller communicate with the repository through a service layer, rather than directly

Reply:
so all the services that were created have their own logic / code in them
but if we create a service for UserRepository (TokenRepository will probably deleted because we decided to not use logout functionality), we really don't have a logic / code to put in them...
we can create a service layer for UserRepository, but it feels like we are creating extra code for the sake of formatting... which feels unnecessary
also, since UserRepository is declared as final, there won't be any changes applied to it, so it should be pretty safe

I personally think it is okay to use UserRepository directly at this point. I can create a service if you want.

Reply:
i think in the security/authorization context, having just the repository is fine. but when we start adding more relationships for the user, it could be beneficial to have that service class to have all the business logic contained within it

Reply:
okay, that makes sense, I'll create the service layer